tests/suites/main_test.function

```diff
@@ -348,7 +348,7 @@ int main(int argc, const char *argv[])
           testfile_index++ )
     {
         int unmet_dep_count = 0;
-        char *unmet_dependencies[20];
+        char *unmet_dependencies[20]; /* only filled when verbose != 0 */
```

Comment:
The comment is not completely true given the change below. Just remove?

Reply:
Remove what? `unmet_dependencies` is used when `verbose != 0` and I don't want to remove it. The code below only fills it if `verbose != 0`; this PR does not change that.

Reply:
I was suggesting to remove/improve the added comment as the code you add at line 398 fills the entry 0 of unmet_dependencies[] with NULL when option_verbose == 0.

Reply:
I don't get it. The comment is true regardless of the change below.